scripts/test.sh

```diff
@@ -0,0 +1,6 @@
+curl -X POST -H "Content-Type: application/json" --data @cvpc.json http://10.213.43.158:9009/project/3dda2801-d675
+-4688-a63f-dcda8d327f50/vpcs
+curl -X POST -H "Content-Type: application/json" --data @csn.json http://10.213.43.158:9009/project/3dda2801-d675-
+4688-a63f-dcda8d327f50/subnets
+curl -X POST -H "Content-Type: application/json" --data @cp1b.json http://10.213.43.158:9009/project/3dda2801-d675
```

Comment:
Need to register nodes if binding host id is not null.

Also I would suggest you to provide similar features as provided in https://github.com/futurewei-cloud/alcor/blob/master/scripts/sampleVpcTest.sh

e.g. use json_pp to show clear json message, allow users to delete vpc/subnet/port etc.

Reply:
fixed in ac8819a for the node register, for similar feature i think we could add them in the new issue

Reply:
@haboy52581 For the scripts that are moved out from this PR, please address the remaining comments left. 

Please check in your change to a feature branch in your folked repo, instead of the master branch. Refer to https://github.com/futurewei-cloud/alcor/blob/master/docs/README.md for development guidance.